### DISCUSSION TITLE
Radio buttons not working properly

### DISCUSSION THREAD

**Discussion by RezaODB** [livewire/livewire#6370] [1. Help] [answered]
In a Livewire 3 project, I have the following issue with 3 radio buttons in my form: I can click on the 3 buttons and they all stay "checked". They behave like checkboxes... Any idea why?  

        <div>
            <input type="radio" class="border-gray-300 mr-4" value="private" id="private" wire:model.live="venue">
            <label for="private">{{ __('txt.venue_private') }}</label>
        </div>
        <div>
            <input type="radio" class="border-gray-300 mr-4" value="public" id="public" wire:model.live="venue">
            <label for="public">{{ __('txt.venue_public') }}</label>
        </div>
        <div>
            <input type="radio" class="border-gray-300 mr-4" value="no" id="no" wire:model.live="venue">
            <label for="no">{{ __('txt.venue_no') }}</label>
        </div>

**Comment by RezaODB** [comment, marked as answer]
I have the answer: contrary to what is written in the documentation: you DO need to add the name attribute!!!

**Reply by edalzell** [reply]
Any chance you could PR that into the docs?

**Reply by RezaODB** [reply]
I have tried, but documentation for V3 is not accessible for editing it appears...

**Reply by joshhanley** [reply]
@RezaODB docs are in the main repo now, see https://github.com/livewire/livewire/tree/main/docs
